src/test/java/kr/allcll/backend/admin/AdminServiceTest.java

```diff
@@ -139,11 +139,11 @@ void disconnectAfterSsePropagate() throws InterruptedException {
     }
 
     @Nested
-    @DisplayName("어드민 첫 조회 기능을 테스트한다.")
+    @DisplayName("어드민 조회 기능을 테스트한다.")
     class adminGetStatus {
 
         @Test
-        @DisplayName("SSE가 연결되었다가 끊어졌을 때 정상 응답을 검증한다.")
+        @DisplayName("SSE가 연결 상태였다가 해제 되었을 때 false 반환을 검증한다.")
```

Comment:
```suggestion
        @DisplayName("SSE 연결을 차단하면 false를 반환한다.")
```
테스트명을 수정해 보면 어떨까요? 위에는 예시에요! 
기존 문장은 하루만 지나도 이해하기 어려울 것 같아요 😢 문장만 봐선 어떤 걸 테스트하는지 파악하기 어렵네요. 

또, 현재 검증하고 있는 부분이 비전공 스케줄링을 멈춘 상황이랑 SSE 연결을 차단한 상황이니까 이게 테스트에 드러나야 할 거 같아요. 
추가로 아래 테스트 명도 모두 수정해 주세요!

Reply:
중도 차단하면 false를 반환한다고 수정햇어요